### PR TITLE
align with app-helpers API

### PR DESCRIPTION
**Pull request by randyaskin**
Switch over to app-helpers API to align with context-browser / tree.

### CODE REVIEW

File: px-breadcrumbs.es6.js [hunk 1]
```diff
@@ -72,224 +64,137 @@
...
-      },
       _hideNoResultLi: {
         type: Boolean,
         value: true
+      },
+      _breadcrumbsObj: {
+        type: Object
       }
     },
-    behaviors: [Polymer.IronResizableBehavior],
```
Comment (benoitjchevalier) [inline px-breadcrumbs.es6.js:90]: Is iron resize still going to be called if you don't include this behavior?

File: px-breadcrumbs.es6.js [hunk 2]
```diff
@@ -72,224 +64,137 @@
...
     observers: [
-      '_calculatePath(_selectedItem)', 
-      '_rebuildBreadcrumbsDisplayOptions(_selectedItemPath, _ulWidth)',
-      'prepareData(breadcrumbData)'
-      ],
+      '_getDisplayMode(_ulWidth, _selectedItemPath, _breadcrumbsObj)'
+    ],
     attached() {
-      document.addEventListener('click', this._onCaptureClick.bind(this));
+      this.fire('iron-resize');
```
Comment (benoitjchevalier) [inline px-breadcrumbs.es6.js]: why do we fire an iron-resize event here? Might notify other people that size has changed when it  might not be time to

File: px-breadcrumbs.es6.js [hunk 3]
```diff
@@ -72,224 +64,137 @@
...
+    * isn't shortened into the container.
     */
     _createArrayWithOverflow(strArray, _ulWidth, breadcrumbsObj) {
-      
+
       var pointer = 0,
           currentAccumSize = breadcrumbsObj.sizeOfAllShortenedItemsExcludingLastItem,
           sizeOfFullLastItem = breadcrumbsObj.sizeOfFullLastItem,
-          sizeOfEllipsis = breadcrumbsObj.sizeOfEllipsis,
+          sizeOfEllipsis = 16,
```
Comment (benoitjchevalier) [inline px-breadcrumbs.es6.js]: why change it to a hard coded value? Also this won't take font into account

Comment (randyaskin) [reply px-breadcrumbs.es6.js]: It uses px-icon now, not text, so it's definitely going to be 16px.

Comment (benoitjchevalier) [reply px-breadcrumbs.es6.js]: isn't that the '...' text?

Comment (randyaskin) [reply px-breadcrumbs.es6.js]: The overflow object uses the "..." icon now:

screen shot 2017-07-28 at 1 00 48 pm

Although I just noticed that sizeOfEllipsis is 16 in one place and 36 in another, that's problematic...

File: px-breadcrumbs.es6.js (hunk 2 above)
Comment (randyaskin) [reply px-breadcrumbs.es6.js]: Something must not have been working while I was developing and that helped it, but it seems to be working without it now.

File: px-breadcrumbs.es6.js (hunk 1 above)
Comment (randyaskin) [reply px-breadcrumbs.es6.js:90]: Not sure what you mean. Don't you have to include the behavior in order to use it?

File: px-breadcrumbs.es6.js (hunk 3 above)
Comment (benoitjchevalier) [reply px-breadcrumbs.es6.js]: Oh ok, I thought that was the ellipsis in a middle of a word

File: px-breadcrumbs.es6.js (hunk 1 above)
Comment (benoitjchevalier) [reply px-breadcrumbs.es6.js:90]: I misread that as the behavior had been deleted, didn't see that it had moved higher up, all good

File: wct.conf.json [hunk 4]
```diff
@@ -3,8 +3,7 @@
   "plugins": {
     "local": {
       "browsers": [
-        "chrome",
-        "firefox"
```
Comment (benoitjchevalier) [inline wct.conf.json:7]: removing firefox on purpose?

Comment (randyaskin) [reply wct.conf.json:7]: All the rest of our components only test in Chrome on local... just aligning this one.

Comment (benoitjchevalier) [reply wct.conf.json:7]: Sweet, didn't realize that, don't think we've migrated vis for that

**Pull request merged**